Lesson-4-(Iteration).md

```diff
@@ -1,4 +1,6 @@
-## In the prep work this week (chapter 9), the students learned:
+# Lesson 4: Iteration
+
+## In the prep work this week (Chapter 9), the students learned:
```

Comment:
remove chapter #? or replace it with chapter name?

Reply:
Removed!